index.md

```diff
@@ -1,6 +1,13 @@
 # We are nmstate!
 A declarative network manager API for hosts.
 
+## Table of Contents
```

Comment:
I find this ToC a bit awkward for such a short document.
I'd prefer to add a sentence under "What is it", saying
"You can see demonstrations of what it can do [here](./demos.md)

Reply:
I sounds nice to add it there, but it will be hard to find IMO.
We will probably add more content on other pages as well, so a ToC will have more meaning with time.

Reply:
Instead of the uneven ToC which I don't like, you can add a section like the following. When you add a ToC, it would be added, there, too.

# Demos
We have [demos](./demos.md) showing how nmstate can be used.